### DISCUSSION TITLE
Request guard `AdminToken` failed: "Unauthorized" - after changed TOKEN from text to argon2 .

### DISCUSSION THREAD

**Discussion by masterwishx** [dani-garcia/vaultwarden#3416] [Q&A]
Using latest version of vaultwarden in docker in Unraid .

After changed TOKEN from plain text to argon2 (vaultwarden hash)
There is some Warnings: 

[WARN] Request guard `AdminToken` failed: "Unauthorized".
[WARN] Responding with registered (admin_login) /admin 401 catcher. 

![image](https://user-images.githubusercontent.com/28630321/229783857-edddaf1f-aa01-45a2-8e73-237b22c15df9.png)

**Comment by stefan0xC** [comment]
As [explained in this discourse post](https://vaultwarden.discourse.group/t/warn-log-messages-when-accessing-the-admin-page/2544/2?u=stefan0xc) [this change](https://github.com/dani-garcia/vaultwarden/pull/2886) has been added in [v1.27.0](https://github.com/dani-garcia/vaultwarden/releases/tag/1.27.0) so this should be independent of the change of the `ADMIN_TOKEN` and it has more to do with upgrading to a newer version.

**Comment by masterwishx** [comment]
I'm using Unraid and auto update every time there is update avaliable, but I didn't enter admin page befor, only now for change admin token to hash...
So it's normal this warning?
